app/src/main/res/values-nl/strings.xml

```diff
@@ -222,5 +222,7 @@
     <string name="generic_alert_title">Waarschuwing</string>
     <string name="generic_exception_message">Onbekende fout. Neem contact op met Yona support.</string>
     <string name="generic_mobile_number_format_error">Ongeldig telefoonnummer. Controleer het formaat</string>
+    <string name="yona_notification_content">Apparaatapp-activiteit wordt gecontroleerd door Yona</string>
```

Comment:
Please make this "App-gebruik wordt gemonitord door Yona"

Reply:
Done.